scripts/linters/pylint_extensions.py

```diff
@@ -829,6 +834,11 @@ def check_docstring_structure(self, node):
         """
         if node.doc:
```

Comment:
Please update this function's docstring to properly show what it does. Also change the description [here](https://github.com/oppia/oppia/blob/183ca3b9706ec5fb1ea1d787b51746db29983198/scripts/linters/pylint_extensions.py#L778-L779) (and other places if you find) which describes how `check_docstring_structure` works.